lib/valueflow.cpp

```diff
@@ -2949,7 +2949,7 @@ struct ValueFlowAnalyzer : Analyzer {
                 addErrorPath(tok, "Assuming container is " + s);
             } else {
                 std::string s = state ? "true" : "false";
-                addErrorPath(tok, "Assuming condition is " + s);
+                addErrorPath(tok, s);
```

Comment:
Why is this removed? `Assuming condition is false` is clearer than just saying `false`.

Reply:
Yea that because the `MultiValueFlowAnalyzer` adds a `Assuming condition is` here: https://github.com/danmar/cppcheck/blob/main/lib/valueflow.cpp#L7220

That should be removed there but kept here, because when we are using the other `SingleValueFlowAnalyzer` it would just be `false` instead of `Assuming condition is false`.